### PR TITLE
Update documentation and feature list to 21.06.

### PR DESCRIPTION
**Pull request by otroan**
Update master documentation link to latest.

Signed-off-by: Ole Troan <ot@cisco.com>

### CODE REVIEW

**Comment by edwarnicke** [comment]
Could you rebase this onto the latest from master?

File: static/docs/vpp/master/_static/documentation_options.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 var DOCUMENTATION_OPTIONS = {
     URL_ROOT: document.getElementById("documentation_options").getAttribute('data-url_root'),
-    VERSION: '20.01',
+    VERSION: 'master',
```
Comment (ayourtch) [inline static/docs/vpp/master/_static/documentation_options.js:3]: is this right that this is master as opposed to 21.06 ?

File: static/docs/vpp/master/about.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
   
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   
-  <title>About &mdash; The Vector Packet Processor 20.01 documentation</title>
+  <title>About &mdash; The Vector Packet Processor master documentation</title>
```
Comment (ayourtch) [inline static/docs/vpp/master/about.html:11]: Same as above - master = what am I missing ?

File: content/vppProject/vppbuilding/vpp.md [hunk 3]
```diff
@@ -10,10 +10,10 @@ btntxt="Developing with VPP "
 # The second is the url
 latest = "Latest Release, /docs/vpp/latest/gettingstarted/developers"
 versions = ["Master, /docs/vpp/master/gettingstarted/developers",
+	 "version 21.06, /docs/vpp/v2106/gettingstarted/developers",
 	 "version 21.01, /docs/vpp/v2101/gettingstarted/developers",
 	 "version 20.09, /docs/vpp/v2009/gettingstarted/developers",
-	 "version 20.05, /docs/vpp/v2005/gettingstarted/developers",
-	 "version 20.01, /docs/vpp/v2001/gettingstarted/developers"]
+	 "version 20.05, /docs/vpp/v2005/gettingstarted/developers"]
```
Comment (dwallacelf) [inline content/vppProject/vppbuilding/vpp.md:16]: Same comment here.  Please remove 20.09 & 20.05

File: content/documentation/vpp.md [hunk 4]
```diff
@@ -7,8 +7,9 @@ The Vector Packet Packet Processor (VPP) is the core technology for FD.io.
 <br>
 To get started with VPP check out [Installing VPP](/vppproject/vppinstalling).
 <br>
-To read the VPP User documents please visit the [VPP Latest Release](/docs/vpp/master),
- [Versions 21.01](/docs/vpp/v2101) or [Versions 20.09LTS](/docs/vpp/v2009).
+To read the VPP User documents please visit the [VPP Latest Release](/docs/vpp/latest),
+[Master](/docs/vpp/master), [Versions 21.06](/docs/vpp/v2106), [Versions 20.09](/docs/vpp/v2009)
+or [20.05](/docs/vpp/v2005).
```
Comment (dwallacelf) [inline content/documentation/vpp.md:12]: The supported release versions include 21.06 & 21.01 per the historical release support policy.
Please remove versions 20.09 and 20.05 and add 21.01 in this list.

**Review by dwallacelf — CHANGES_REQUESTED** [review]

**Review by dwallacelf — CHANGES_REQUESTED** [review]
content/documentation/vpp.md, Line 16:  "The source code documentation is at [v21.01](https://docs.fd.io/vpp/21.01)."

s/01/06/g

File: static/docs/vpp/master/_static/documentation_options.js (hunk 1 above)
Comment (otroan) [reply static/docs/vpp/master/_static/documentation_options.js:3]: This patch builds two document trees. One for master, one for 21.06. static/docs/vpp/master ...

File: static/docs/vpp/master/about.html (hunk 2 above)
Comment (otroan) [reply static/docs/vpp/master/about.html:11]: Same as above

**Pull request merged**